README.md

```diff
@@ -97,12 +97,22 @@ var sleepMs = configWrapper.Get<int>("sleep-time-in-ms", 1000);
 Loads from the Windows registry, defining a root key for my application.
 
 ``` 
-IWritableConfigWrapper = new WindowsRegistryConfigWrapper("HKLM/MyApplication/");
+IWritableConfigWrapper = new WindowsRegistryConfigWrapper("HKLM/Software/MyApplication/");
```

Comment:
I was thinking here to use something like "ConfigWrapper" as the test case does "HKCU/ConfigWrapper" instead of "MyApplication" but not sure if needed to much here.